### PR TITLE
Fixed: (Avistaz Family) Age Parsing

### LINKED ISSUE #1212: Update avistaz network age logic
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Prowlarr assumes times from sites are UTC which causes issues with age calculation.

### Describe the solution you'd like

avistaz sites per staff are UTC-5;  prowlarr should treat the dates/times accordingly and calculate appropriately.

### Describe alternatives you've considered

N/a

### Anything else?

> I am not 100% sure what the exact timezone is, but I think UTC-5 is correct, as that should be approx where the tracker server is located.

[AB#4177](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/4177)

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
Pre Fix
image
Post Fix
image
image

#### Todos
- [x] Tests
- Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #1212

### CODE REVIEW

**Comment by bakerboy448** [comment]
tests pass fine - azure being azure....

**Comment by Qstick** [comment]
> tests pass locally - azure failing for alpine for no rationale reason

The rational reason is that the code you added doesn't work on alpine. Likely due to how you implemented the timezone parsing.

File: src/NzbDrone.Core/Indexers/Definitions/Avistaz/AvistazParser.cs [hunk 1]
```diff
@@ -14,6 +14,9 @@ public class AvistazParser : IParseIndexerResponse
     {
         private readonly HashSet<string> _hdResolutions = new HashSet<string> { "1080p", "1080i", "720p" };
 
+        // Servers are Eastern US Time Zone
+        private readonly TimeZoneInfo _avistazTimeZone = TimeZoneInfo.FindSystemTimeZoneById("US Eastern Standard Time");
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Definitions/Avistaz/AvistazParser.cs:18]:
```suggestion
        private readonly TimeZoneInfo _avistazTimeZone = TimeZoneInfo.FindSystemTimeZoneById("America/New_York");
```

Use official IANA timezone to ensure this works on more OSs

**Comment by Qstick** [comment]
The other option is we append TZ data to the incoming string we parse given FindSystemTimeZoneById can vary based on on OS culture data. 

We'll see if this works, but sometime in mid-late net6 the time zone discrepancies between OS's were supposed to be resolved so that `FindSystemTimeZoneById ` could parse either on either OS (Windows vs Posix)

**Comment by bakerboy448** [comment]
> The other option is we append TZ data to the incoming string we parse given FindSystemTimeZoneById can vary based on on OS culture data.
> 
> We'll see if this works, but sometime in mid-late net6 the time zone discrepancies between OS's were supposed to be resolved so that `FindSystemTimeZoneById ` could parse either on either OS (Windows vs Posix)

supposedly fixed in the previews
https://devblogs.microsoft.com/dotnet/date-time-and-time-zone-enhancements-in-net-6/

**Comment by bakerboy448** [comment]
BSD now failing due to node `This step requires a node version that does not exist in the agent filesystem. Path: /usr/home/azure/prowlarr/externals/node16/bin/node`

**Pull request merged**